shepherd_bms/include/datastructs.h

```diff
@@ -1,5 +1,67 @@
 #ifndef DATASTRUCTS_H
 #define DATASTRUCTS_H
 
+#include <nerduino.h>
+
+
+//make data types unsigned int 16 bit
+
+/*individual segment data*/
```

Comment:
could you change to "individual chip data" or something, since there are 2 chips per segment board